src/custom/ecospheres/views/HomeView.vue

```diff
@@ -31,94 +36,320 @@ useHead({
+                      alt=""
+                      class="illustration fr-mx-auto block"
+                      width="64"
+                      height="64"
+                    />
+                  </div>
+                  <div class="fr-col-md-9">
+                    <h3>Centraliser les données territoriales</h3>
+                    <p>
+                      Les données territoriales sont collectées, regroupées et
+                      catégorisées par cas d’usage sous la forme de bouquets.
+                    </p>
+                  </div>
+                </div>
+              </li>
+            </ul>
           </div>
-          <div class="fr-col-md-6 fr-hidden fr-unhidden-md fr-grid-row--center">
-            <img
-              src="../assets/home-bouquets-schema.svg"
-              alt=""
-              loading="lazy"
-              class="illustration"
-              width="441"
-              height="290"
-            />
-          </div>
-        </div>
-        <div class="fr-mt-10v">
-          <h3>Trouvez un bouquet par thématique</h3>
-          <HomeThemes v-if="config.themes" />
         </div>
       </div>
     </section>
-    <section v-if="searchConfig.display" class="fr-container fr-py-12v">
-      <h2>Jeux de données</h2>
-      <p>
-        Recherchez parmi les données présentes sur <i>ecologie</i
-        ><strong>.data.gouv</strong><i>.fr</i>.
+    <section class="fr-container fr-py-12v">
+      <h2>Les bouquets à découvrir</h2>
+      <ul class="fr-grid-row fr-grid-row--gutters" role="list">
+        <li
+          v-for="topic in lastTopics"
+          :key="topic.id"
+          class="fr-col-12 fr-col-xl-4"
+        >
+          <BouquetCardHome :bouquet="topic" />
+        </li>
+      </ul>
+      <p class="fr-m-0">
+        <a
+          href="/bouquets"
+          class="fr-link fr-icon-arrow-right-line fr-link--icon-right"
+          >Voir tous les bouquets</a
+        >
       </p>
-      <div class="fr-grid-row fr-grid-row--gutters">
-        <div class="fr-col-12 fr-col-md-8">
-          <DsfrSearchBar
-            v-model="query"
-            button-text="Rechercher"
-            class="search-bar-input"
-            :label="searchConfig.placeholder"
-            placeholder=""
-            :large="true"
-            @search="doSearch"
-          />
-        </div>
+    </section>
+    <section class="fr-container--fluid faq">
+      <div class="fr-container fr-py-16v faq">
+        <h2>Foire aux questions</h2>
+        <HomeFaq />
       </div>
     </section>
   </div>
 </template>
 
 <style scoped>
-.bouquets {
+.bouquets,
+.faq {
   background-color: #f6f6f6;
 }
 .illustration {
   max-width: 100%;
 }
+.organize-data {
+  background-color: white;
+  padding: 32px 0;
+
+  li {
+    border-right: 1px solid var(--border-default-grey);
+    padding-right: 1rem;
+  }
+
+  li:last-child {
+    border: none;
+  }
+
+  @media (max-width: 992px) {
+    padding: 16px;
```

Comment:
Règle sans sélecteur.

```suggestion
    & {padding: 16px;}
```

Reply:
Ton IDE affiche un warning ? Chez moi c'est bon

Reply:
Pas à proprement parler non, ça fonctionne mais ça me fait bizarre de voir une règle sans sélecteur 😄 

![image](https://github.com/user-attachments/assets/6e031053-f684-4550-8ba0-d4a6057d223f)